woocommerce/woocommerce
# [CYS] Use outline instead of border, so the border width does not affect the card size

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR fixes the flickering effect on the font pairing cards in the assembler.
The card height was calculated with a ratio that depended on the initial card height and width, which was affected by the border width. Using `outline` instead of `border` makes the size of the card not depend on the border width, avoiding the weird flickering effect.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure the `WooCommerce Beta Tester` plugin is installed and activated (available on this monorepo).
2. Ensure you install the latest version of Gutenberg.
3. Head over to `/wp-admin/tools.php?page=woocommerce-admin-test-helper` and enable `customize-store` feature flag.
4. Go to `WooCommerce > Home`, click on `Start customizing`, and finish the flow.
5. Go to the `Change fonts` section, hover over the different font pairings and check there's no change in the height of the cards.

| Before | After |
|--------|--------|
| https://github.com/woocommerce/woocommerce/assets/186112/46484429-8684-4751-8e6c-c01e40710a7d |https://github.com/woocommerce/woocommerce/assets/186112/530ae221-e1ff-448f-a2ab-4cafdbea4561 | 


### Changelog entry




<details>

#### Significance




#### Type




#### Message 
CYS - Fix the flickering effect on hover on the font pairing cards.
#### Comment 

</details>
